server/tests/interpreter_test.js

```diff
@@ -1066,22 +1069,26 @@ exports.testNetworking = async function(t) {
       CC.connectionUnlisten(8888);
    `;
   initFunc = function(intrp) {
-    intrp.addVariableToScope(intrp.global, 'recieve', intrp.createAsyncFunction(
-        'recieve', function(resolve, reject) {
-          var reply = '';
-          // Recieve some data from the server.
-          var client = net.createConnection({ port: 8888 }, function() {
-            client.on('data', function(data) {
-              reply += data;
-            });
-            client.on('end', function() {
-              resolve(reply);
-            });
-            client.on('error', function() {
-              reject();
-            });
+    intrp.addVariableToScope(intrp.global, 'recieve', new intrp.NativeFunction({
+      name: 'recieve', length: 0,
+      call: function(intrp, thread, state, thisVal, args) {
+        var reply = '';
+        var rr = intrp.getResolveReject(thread, state);
+        // Recieve some data from the server.
```

Comment:
I before E except after C.

Reply:
Fixed.  (So, you *are* looking at the tests…)